doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ppr.rst

```diff
@@ -0,0 +1,175 @@
+.. _ug_nrf54h20_ppr:
+
+Working with the PPR core
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The PPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *Peripheral Processor* (PPR).
+The PPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API.
+These peripherals have IRQs routed to PPR:
+
+* ##TODO
+
+All other peripherals available to the application core can also be used with PPR.
+However, they require the use of *polling mode*.
```

Comment:
I'm not sure if this is relevant to PPR. Even though it's technically possible, PPR is too slow to poll USB or CAN. It wouldn't be any practical application